primitives/src/policy.rs

```diff
@@ -105,6 +105,12 @@ impl Policy {
     /// in proof-of-work.
     pub const VALIDATOR_DEPOSIT: u64 = 10_000_000_000;
 
+    /// The stake necessary to create a staker in Lunas (1 NIM = 100,000 Lunas).
+    /// This minimum is applied to the amount of:
+    ///     - non retired stake
+    ///     - total stake balance
+    pub const MINIMUM_STAKE: u64 = 10_000_000;
```

Comment:
Offtopic comment:
I always find this way of separating by thousands weird, because 10k luna doesn't tell me anything useful. I would prefer this whole file separates by 5 zeros at the end first, and then the NIM part by thousands:
```suggestion
    pub const MINIMUM_STAKE: u64 = 100_00000;
```

Reply:
I guess it'd need a separate PR, but I generally agree.